### ISSUE TITLE
Disconnect if PublishOnHistoryAdd is set to false

### ISSUE THREAD

**Issue by masterada** [centrifugal/centrifuge#119]
With v0.8.0, if PublishOnHistoryAdd is set to false (it was set to false by mistake), i keep getting disconnected with reason `insufficient state`. I think this is because the offset magic:
```
if ARGV[4] ~= '' then
	local payload = "__" .. offset .. "__" .. ARGV[1]
	redis.call("publish", ARGV[4], payload)
```
is missing from engine_redis.Publish

**Comment by FZambia** [comment]
@masterada thanks for pointing on this bug! Fix in related pr.

**Comment by FZambia** [comment]
released with v0.8.1
